tests/torch/quantization/test_onnx_export.py

```diff
@@ -193,3 +196,26 @@ def test_target_compression_idx(tmp_path):
                                                                                 onnx_graph)
     assert input_low_t_attr.shape == (1, TargetCompressionIdxTestModel.CONV2D_TRANSPOSE_TARGET_CHANNEL_COUNT, 1, 1)
     assert input_low_t_attr.shape == input_high_t_attr.shape
+
+
+def test_are_quantized_weights_exported_correct(tmp_path):
+    model = TwoConvTestModel()
+    nncf_config = get_config_for_export_mode(False)
+    nncf_config.update({"input_info": {
+        "sample_size": [1, 1, 20, 20]
+    }})
+    fill_params_of_model_by_normal(model)
+    _, compression_ctrl = create_compressed_model_and_algo_for_test(model, nncf_config)
+
+    quantizers = compression_ctrl.weight_quantizers.values()
```

Comment:
Missing the part where quantized weights are filling by middle quants value. Could you please point out, where it happens

Reply:
I see, it's in `test_overflow_issue_export.py`, but then I don't understand a point of this test. Is middle quant initialization isn't enough? It would be clearer for me if middle quant initialization was an option in such export test, so all kinds of an export test was in one place

Reply:
Tests from `test_overflow_issue_export.py` and `test_onnx_export.onnx` have been squeezed into one